Code/Unsupervised_reconstruction/lm/PriorLM.py

```diff
@@ -213,7 +213,7 @@ def training(self, data: str, mini_batch_size: int = 32, epochs: int = 10, save_
 
         adjust_seq_lengths: Callable[[Tensor, Tensor, int], tuple[Tensor, Tensor]] = lambda x, l, _: (x, l+1)
         training_data = [adjust_seq_lengths(
-            *computeInferenceData(tData)) for tData in indices_tensor.split(mini_batch_size, dim=1)]
+            *computeInferenceData_Samples(tData)) for tData in indices_tensor.split(mini_batch_size, dim=1)]
```

Comment:
Wouldn't there be an operation to speed up the procedure? Especially to avoid the need for `adjust_seq_lengths` through a Torch method?

Reply:
The module is not updated with the last changes (for instance in the `inference` method and a review will have to be done).